CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+- Added the character count component. [#61](https://github.com/blackbaud/skyux-forms/pull/61)
+- Added a CONTRIBUTING.md file. [#64](https://github.com/blackbaud/skyux-forms/pull/64)
+- Fixed the file drop component's trash icon. [#62](https://github.com/blackbaud/skyux-forms/pull/62)
```

Comment:
Is this correct? Looking at the PR, it looks like the changes are to the files for the file item component, not the file drop component.

Reply:
ah you're correct, it is the file item. I'll fix.